build-tools/docker/Dockerfile.builder

```diff
@@ -0,0 +1,8 @@
+FROM rust AS builder
+
+WORKDIR /usr/src/
+
+COPY . .
+
+ARG NUM_JOBS=1
```

Comment:
At this moment it's impossible to specify this value when using `build.py`. I'd be nice if the latter had an additional parameter for this.